modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/nhttp/HttpCoreNIOSender.java

```diff
@@ -153,12 +153,9 @@ public void init(ConfigurationContext cfgCtx, TransportOutDescription transportO
 
         connpool = new ConnectionPool();
 
-        proxyConfig = new ProxyConfigBuilder().parse(transportOut).build();
-        if (log.isInfoEnabled() && proxyConfig.getProxy() != null) {
-            log.info("HTTP Sender using Proxy " + proxyConfig.getProxy() + " bypassing " + 
-                proxyConfig.getProxyBypass());
-        }
-        
+        proxyConfig = new ProxyConfigBuilder().build(transportOut);
+        log.info(proxyConfig.logProxyConfig());
+
         Parameter param = transportOut.getParameter("warnOnHTTP500");
```

Comment:
Yeah you have a point here. But IMO we should not fix all the issues we see in a class, when we are doing some specific fix/implementation. That will make things hard when it comes to tracking. For an example even I see formatting issues in the class, I won't correct those stuff, as I don't want add those changes to my PR. If I do so, it will make things hard for someone to keep track of the effective changes made by the PR.